packages/terra-dev-site/src/webpack/templates/bootstrap.template

```diff
@@ -0,0 +1,43 @@
+<% sideEffectImportFilePaths.forEach(sideEffect => print(`import '${sideEffect}';\n`)); %>
```

Comment:
This template was renamed from entry.template to bootstrap.template

no other changes were made.